File: app/src/main/java/fr/masciulli/drinks/net/WebApi.java
package fr.masciulli.drinks.net;

import fr.masciulli.drinks.core.Drink;
import fr.masciulli.drinks.core.Liquor;
import retrofit2.http.GET;
import rx.Observable;

import java.util.List;

public interface WebApi {
    @GET("/api/Drinks")
    Observable<List<Drink>> getDrinks();

    @GET("/api/Liquors")
    Observable<List<Liquor>> getLiquors();
}

File: core/src/main/java/fr/masciulli/drinks/core/WebApi.kt
package fr.masciulli.drinks.core

import retrofit2.http.GET
import rx.Observable

interface WebApi {
    @get:GET("/api/Drinks")
    val drinks: Observable<List<Drink>>

    @get:GET("/api/Liquors")
    val liquors: Observable<List<Liquor>>
}
